src/tv2_afvd_showstyle/migrations/util.ts

```diff
@@ -105,3 +107,98 @@ export function getOutputLayerDefaultsMigrationSteps(versionStr: string): Migrat
 		})
 	)
 }
+
+/**
+ * "Renames" the id of a table by copying over all values over into a new table which has the new id - then removes the values from the old table
+ */
+export function renameTableId(version: string, oldTableId: string, newTableId: string): MigrationStepShowStyle {
+	return {
+		id: `${version}.rename.table.id.${oldTableId}.to.${newTableId}`,
+		version,
+		canBeRunAutomatically: true,
+		validate: (context: MigrationContextShowStyle) => {
+			const oldConfigTable = (context.getBaseConfig(oldTableId) as unknown) as TableConfigItemValue
+			if (!oldConfigTable || oldConfigTable.length === 0) {
+				return false
+			}
+
+			const newConfigTable = (context.getBaseConfig(newTableId) as unknown) as TableConfigItemValue
+			return !newConfigTable
+		},
+		migrate: (context: MigrationContextShowStyle) => {
+			const oldConfigTable = (context.getBaseConfig(oldTableId) as unknown) as TableConfigItemValue
+			const newConfigTable = ((context.getBaseConfig(newTableId) as unknown) as TableConfigItemValue) ?? []
+			oldConfigTable.map(value => newConfigTable.push(value))
+
+			context.setBaseConfig(newTableId, newConfigTable)
+			context.setBaseConfig(oldTableId, [])
+		}
+	}
+}
+
+/**
+ * "Renames" the id of a column by overriding the value of the old column onto the new column - then removes the value from the old column
+ */
+export function renameColumnId(version: string, oldColumnId: string, newColumnId: string): MigrationStepShowStyle {
```

Comment:
Why is _Column_ in the name of this function? You're using it to rename `SelectedGraphicsSetupName` which is neither a column, nor a table, it's just a Show Style (Blueprints) Config. An analogous function for Studios is `RenameStudioConfig`. Same applies to `renameColumnIdForAllVariants`

Reply:
Good point. I'm not so sure why I went with that naming. My best guess is that I looked at `renameTableColumn` for inspiration to write the migration and then didn't think far enough to see this wasn't a column.

Will update

Reply:
Updated